src/order.html

```diff
@@ -0,0 +1,26 @@
+---
+layout: default.html
+---
+
+<h2>Order</h2>
+<p>Please use this to order biscuits!</p>
+<form action="/action_page.php">
+    <fieldset>
+      <legend>Order form:</legend>
+      <label for="name">Name:</label><br>
+      <input type="text" id="name" name="name" value="John"><br>
```

Comment:
Avoid using `br` and instead rely on CSS to control your form elements' look and structure.